src/main/java/za/ac/cput/school_management_grp33/domain/location/Address.java

```diff
@@ -0,0 +1,8 @@
+package za.ac.cput.school_management_grp33.domain.location;
+/*
+Address.Java;
+Author: Kevin Lionel Mombo Ndinga(218180500)
+Date: 11 of June 2022;
+ */
+public class Address {
+}
```

Comment:
this belongs in the lookup package, not in the location

Reply:
ok great i will change it now